docs/list-available-operators.md

```diff
@@ -0,0 +1,31 @@
+# How do I list Operators available to install?
+
+There is an extension API in OLM named `PackageManifest` that contains information existing `CatalogSources`, which is essentially a repository of CSVs, CRDs, and packages that define an application in the cluster. By querying that API, you can see the list of available operators.
+
+There are two different types of `CatalogSource` in OLM: global and namespace `CatalogSource`. The global `CatalogSource` contains operators that will be available for all namespaces while namespace `CatalogSource` only contains operators that are only available for a specific namespace.
+
+## PackageManifest Commands
+
+You can use these example commands via either OpenShift CLI (oc) or kubectl CLI (kubectl) to list available operators in a specific namespace. `PackageManifest` will return the union of global, which are available globally, and namespaced operators in namespace you're requesting.
+
+```bash
+$ oc get packagemanifest -n <namespace>
+```
```

Comment:
I don't believe that olm-book is supposed to include OpenShift specific commands.